pkgs/applications/editors/qemacs/default.nix

```diff
@@ -9,12 +15,42 @@ stdenv.mkDerivation rec {
     sha256 = "156z4wpj49i6j388yjird5qvrph7hz0grb4r44l4jf3q8imadyrg";
   };
 
-  buildInputs = [ libpng libX11 libXext libXv ];
+  buildInputs = lib.optionals enableX11 [ libpng libX11 libXext libXv ];
+
```

Comment:
removed unneeded `configurePlatforms = [ ];` , the build system just ignores them

Reply:
> removed unneeded `configurePlatforms = [ ];` , the build system just ignores them

It was probably needed for https://github.com/NixOS/nixpkgs/pull/209418; I wrote both PRs and then tried to separate them into two commits.